src/components/AgendaItems/AgendaItemsUpdateModal.tsx

```
@@ -9,6 +9,65 @@ import convertToBase64 from 'utils/convertToBase64';
 import styles from './AgendaItemsContainer.module.css';
 import type { InterfaceAgendaItemCategoryInfo } from 'utils/interfaces';
 
+import React from 'react';
```

Biome

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)